hadoop-ozone/dist/src/main/smoketest/s3/boto_client.py

```diff
@@ -34,7 +34,9 @@ class TestBotoClient(unittest.TestCase):
     setup_done = False
     target_bucket = None
     ozone_endpoint_url = None
- 
+    random_bucket_name = ''.join(
```

Comment:
Can we prefix them with ozonetest? This would help discover buckets created by the tests.

Reply:
Good suggestion!  Will do.